### PR TITLE
[LPS-146936] [TestRay] Created TestRay Compare Runs

### PR DESCRIPTION
**Pull request by iagochaves**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/CompareRuns/index.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	const {colors, columns, data} = useTableChartData();
+
+	return (
+		<Container title={i18n.translate('details')}>
+			<div className="d-flex flex-wrap">
+				<div className="col-8 col-lg-8 col-md-12">
+					<QATable
+						items={[
+							{
+								title: `${i18n.translate('run')} A`,
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/CompareRuns/index.tsx:33]: No need to use `${}` here

title: `i18n.translate('run')`

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Comment by iagochaves** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**